tests/Integration/PrestaShopBundle/Controller/GridControllerTestCase.php

```diff
@@ -128,12 +128,18 @@ protected function parseEntitiesFromGridTable(Crawler $crawler): TestEntityDTOCo
             }
         }
 
+        // Get total number
+        $gridTitleHeader = trim($crawler->filter('.card-header-title')->first()->text());
+        preg_match('/\(([0-9]+)\)/', $gridTitleHeader, $titleMatches);
+        $totalCount = (int) ($titleMatches[1] ?? 0);
```

Comment:
it all seems so complicated and unreliable. But I guess it is how those tests must be written? :smile: Have no experience in this area :(

Reply:
Well I didn't find any proper data attributes to fetch this data 😅 So I had to parse the title, but you're right it's not ideal I'm gonna add a proper data attribute in the template, it could be useful for JS as well someday